### PR TITLE
Use master as source [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
Hi @Mizux This would be the way to test whatever you have on master. 
You can just restart the PR by adding the following comment to this PR: `@conda-forge-admin, please restart ci`.

I didn't update the deps yet as I wasn't sure what they are going to be. Can you post any change here and I'll update it?

I can also give you some instructions on how you can do that yourself but this seemed easier – let me know if you still want them.

**Comment by Mizux** [comment]
@BastianZim can you test with this diff ?
```diff
%git diff -u origin/main 
diff --git a/recipe/meta.yaml b/recipe/meta.yaml
index 1d790cb..fae5041 100644
--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,10 +1,10 @@
 {% set name = "or-tools" %}
 {% set package_name = "ortools" %}
-{% set version = "9.2" %}
+{% set version = "9.4" %}
 {% set home = "https://developers.google.com/optimization" %}
 {% set license = "Apache-2.0" %}
 {% set license_family = "Apache" %}
-{% set license_file = "LICENSE-2.0.txt" %}
+{% set license_file = "LICENSE" %}
 {% set dev_url = "https://github.com/google/or-tools" %}
 
 
@@ -13,13 +13,13 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/google/or-tools/archive/v{{ version }}.tar.gz
-  sha256: 5337935ea1fa010bb62cf0fc8bedd6de07dda77bff3db7a0f6a36c84c7bd58db
+  git_url: https://github.com/google/or-tools.git
+  git_tag: master
 
 # The meta package
 
 build:
-  number: 3
+  number: 0
   skip: true  # [not linux]
 
 outputs:
@@ -39,6 +39,7 @@ outputs:
       host:
         - abseil-cpp
         - libprotobuf
+        - re2
         - zlib 1.2.11
         # coin or deps
         - coin-or-utils
```
see: https://github.com/Mizux/or-tools-feedstock/commits/main

note: re2 seems to be available, so I would like to test if or-tools correctly manage to find it ;)
ref: https://github.com/conda-forge/re2-feedstock

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
Have added the dif, let's see if it works :)

> note: you can also reset hard your [test-master] branch on my branch main

I can do that but it might be easier if you just fork this repo and apply it yourself and make a PR, then you don't need to wait for me? Whichever is easier for you. The CI runs on every PR so you can just try it.
You just need to make sure to post the rerender comment above so https://github.com/conda-forge/or-tools-feedstock/pull/15#issuecomment-1102721693 that the rest of the infrastructure gets updated accordingly, if you change the deps etc. 
Basically, the meta.yaml file is the instruction for how to build the package and the actual steps are in the other files which will sometimes get updated when you rerender.

> re2 seems to be available, so I would like to test if or-tools correctly manage to find it ;)

Also added it :)
